code/game/objects/items/plushes.dm

```diff
@@ -696,6 +721,84 @@
 	desc = "Hgrgrhrhg cute."
 	icon_state = "flushplush"
 
+/obj/item/choice_beacon/radial/plushie
+	name = "plushie delivery beacon"
+	desc = "Summon your new friend!"
+	icon_state = "gangtool-plushie"
+	var/static/list/plushie_list
+
+/obj/item/choice_beacon/radial/plushie/Initialize(mapload)
+	. = ..()
+	plushie_list = list(
+		/obj/item/toy/plush/bubbleplush,
+		/obj/item/toy/plush/carpplushie,
+		/obj/item/toy/plush/plushvar,
+		/obj/item/toy/plush/narplush,
+		/obj/item/toy/plush/snakeplushie,
+		/obj/item/toy/plush/lizardplushie,
+		/obj/item/toy/plush/slimeplushie,
+		/obj/item/toy/plush/nukeplushie,
+		/obj/item/toy/plush/awakenedplushie,
+		/obj/item/toy/plush/beeplushie,
+		/obj/item/toy/plush/crossed,
+		/obj/item/toy/plush/rouny,
+		/obj/item/toy/plush/runtime,
+		/obj/item/toy/plush/flushed,
+		/obj/item/toy/plush/gondola,
+		/obj/item/toy/plush/moth/atlas,
+		/obj/item/toy/plush/moth/bluespace,
+		/obj/item/toy/plush/moth/brown,
+		/obj/item/toy/plush/moth/clockwork,
+		/obj/item/toy/plush/moth/deadhead,
+		/obj/item/toy/plush/moth/firewatch,
+		/obj/item/toy/plush/moth/gothic,
+		/obj/item/toy/plush/moth/lovers,
+		/obj/item/toy/plush/moth/luna,
+		/obj/item/toy/plush/moth/monarch,
+		/obj/item/toy/plush/moth/moonfly,
+		/obj/item/toy/plush/moth/plasmafire,
+		/obj/item/toy/plush/moth/poison,
+		/obj/item/toy/plush/moth/punished,
+		/obj/item/toy/plush/moth/ragged,
+		/obj/item/toy/plush/moth/rainbow,
+		/obj/item/toy/plush/moth/redish,
+		/obj/item/toy/plush/moth/rosy,
+		/obj/item/toy/plush/moth/royal,
+		/obj/item/toy/plush/moth/snow,
+		/obj/item/toy/plush/moth/whitefly,
+		/obj/item/toy/plush/moth/witchwing
+	)
+
+/obj/item/choice_beacon/radial/plushie/generate_options(mob/living/M)
```

Comment:
single-letter var names hurt :(

Reply:
This is currently standard practice in the codebase when declaring mobs, for better or worse.

Reply:
doesn't mean we can't do better with new code :)

Reply:
I am actively against changing this standard unless it is standardized across the board like it currently is.